solution-base/build.sh

```diff
@@ -34,6 +34,8 @@ KAFKA_OPERATOR_PATH=${ISO_ROOT}/deploy/kafka.yaml
 KUBEDB_OPERATOR_PATH=${ISO_ROOT}/deploy/kubedb.yaml
 KUBEDB_CATALOGS_PATH=${ISO_ROOT}/deploy/kubedb-catalogs.yaml
 ZOOKEEPER_OPERATOR_PATH=${ISO_ROOT}/deploy/zookeeper.yaml
+MONGODB_SINGLE_NODE_PATH=${ISO_ROOT}/deploy/mongodb-1-node.yaml
+MONGODB_THREE_NODE_PATH=${ISO_ROOT}/deploy/mongodb-3-node.yaml
```

Comment:
How about just including the chart in the archive and deploying from the installer? Helm3 is tiller-less so the end result would be identical. The issue with number of nodes presets is that we'd need to keep updating each time there's a new configuration to support.